### ISSUE TITLE
2024-07-12 07:37:10,958 - mmdet - INFO - Epoch [12][45200/45220]	lr: 1.000e-06, eta: 0:00:02, time: 0.099, data_time: 0.007, memory: 3716, loss_relation: nan, relation.recall@20: 13.1938, loss: nan, grad_norm: nan 2024-07-12 07:37:13,171 - mmdet - INFO - Saving checkpoint at 12 epochs

### ISSUE THREAD

**Issue by hanzefang** [franciszzj/VLPrompt#3]
Thanks for your reply. And I have a question: when I train 7 epoch, the loss is nan, why? Or, can you provide you checkpoint?

**Comment by franciszzj** [comment]
Can you try training with all the detection losses turned off and only train on the relation losses to see if the issue of the loss becoming NaN still occurs? I haven't encountered this issue before on my end.

**Comment by hanzefang** [comment]
Thank you. Maybe I use one gpu?
CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --nproc_per_node=1 --master_port=27500 tools/train.py /home/ubuntu/hzf/hzf/scene-graph-generation/VLPrompt-main/configs/psg/baseline_swin_b.py --auto-resume --no-validate --launcher pytorch

**Comment by franciszzj** [comment]
I think this may be the reason for the problem with training loss. Because you only use 1 GPU, the default batch_size_each_gpu is 1, which is too small. It is recommended to use 4 GPUs for experiments.
